### DISCUSSION TITLE
The 'ducking' effect will be applied even if the sound is muted when switching rulesets

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#28984] [Q&A] [answered]
https://github.com/user-attachments/assets/a13ebfb8-20cc-4cb7-ac13-53c6ec584e31

I don't know what to think about this, but it would be weird if I had to turn off the sound effects when I was using my ipad+airpods to play

**Comment by peppy** [comment]
Our eventual goal is to split out hitsounds from effects volume, which would likely solve your underlying issue.

But I guess this is still weird for people that choose to turn off UI sounds (should we even allow that?).

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/29502
